open-source-software-submissions/editor-in-chief-guide.md

```diff
@@ -1,19 +1,131 @@
-- Raises scope/overlap issue with all editors if they see an ambiguous case. This may also be done by handling editors (see below). To initiate discussion, this is posted to the pyOpenSci Slack software review channel, tagging all editors.
-- Responds to pre-submission inquiries and `meta` issues posted to the software-review and software-review-meta repos, similarly pinging channel if discussion is needed. Any editor should feel free to step in on these. See [this section](#responding-to-out-of-scope-submissions) about how to respond to out-of-scope (pre-) submissions.
-- Responds to referrals from JOSS or other publication partners.
-- Monitors the review process pace and reminds other editors to move packages along as needed.
+If the EiC needs to step down at any time before the agreed-upon 
+duration of their tenure, pyOpenSci requests and appreciates 2 weeks notice
+to support finding a new EiC.
 
-## Editor in Chief Checklist
-- Tag issue with `1/editor-checks` tag and assign a main editor.
+### The Editor in Chief fulfills the following roles:
+
+- Watches all issues posted to the software-review repo (either subscribe to repo notifications on GitHub, or watch the `#software-review-feed` channel on Slack).
+- Performs an initial set of editor checks on new package submissions (see template for these checks below)
+- Raises scope/overlap issue with all editors if they see an ambiguous submission. 
+This may also be done by handling editors if appropriate (see note below). If 
+the scope of a package is in question, the EiC should post to the pyOpenSci 
+Slack `software-review` channel, tagging all editors.
+- Assigns package submissions to other editors (including themselves if they have bandwidth), to handle. Assigning may be based upon domain background or just who has bandwidth to lead the review. 
+- Responds to pre-submission inquiries posted to the software-review repository 
+and similarly pings editors in the `software-review slack channel` if discussion 
+is needed. Any editor should feel free to communicate with package authors as it 
+makes sense. See [this section](#responding-to-out-of-scope-submissions) about 
+how to respond to out-of-scope (pre-) submissions.
+- Responds to review referrals from JOSS or other publication partners.
+- Monitors pace of review process and reminds other editors to move packages along as needed.
+- Requests a new EiC when their rotation is up (set a calendar reminder ahead of your expected end date and ask for volunteers in the editors’ Slack channel)
+
+This Editor in Chief position rotates between different pyOpenSci editors. 
+
+```{note}
+In some cases, if the EiC is unable to support a particular package due to 
```

Comment:
Should this language be stronger?

Not making a suggestion since it would span several lines, but:
> If the EiC is unable to support a particular package due to a conflict of interest, they should assign an editor to perform initial checks. The Editor in Chief may also opt to assign an editor to perform checks if they believe that editor is better suited to assess the scope and readiness of a package to be reviewed